packages/server/server.py

```
@@ -0,0 +1,249 @@
+                info_dict=dict(
+                    id=video_id,
+                    title=title,
+                ),
+                total_bytes=os.path.getsize(OUTPUTS_DIR / path),
+                status="completed",
+            ))
+    return flask.jsonify(error=False, data=data)
+
+@app.command()
+def serve(
+    host: str = typer.Option("0.0.0.0", help="Host to bind to"),
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)